test/uk/gov/hmrc/tai/repositories/EmploymentRepositorySpec.scala

```diff
@@ -1288,7 +1303,7 @@ class EmploymentRepositorySpec extends BaseSpec {
           npsConnector = mockNpsConnector,
           employmentBuilder = mockEmploymentBuilder)
 
-      sut.employment(nino, 2).futureValue mustBe Right(employmentWithUnavailableAnnualAccount)
+      sut.employment(nino, 1).futureValue mustBe Right(employmentWithUnavailableAnnualAccount)
```

Comment:
Why has this changed from 2 -> 1?

Reply:
I have changed the sequence number in Json stubs and for key "0-0-0" to 1. And interanally it uses  below
employmentById(id: Int): Option[Employment] = employments.find(_.sequenceNumber == id). So to match provided id with sequence number I have changed it to 1.